backend/src/test/kotlin/app/ehrenamtskarte/backend/auth/webservice/ApiTokenServiceTest.kt

```diff
@@ -0,0 +1,211 @@
+            assertTrue(tokenExists)
+
+            ApiTokenService().deleteApiToken(1, mockDfe)
+
+            val tokenNoLongerExists = ApiTokens.select { ApiTokens.id eq 1 }.singleOrNull() == null
+            assertTrue(tokenNoLongerExists)
+        }
+    }
+
+    @Test
+    fun deleteApiToken_deletesFromOtherAdminTokenSuccessfully() {
+        every { mockJwtPayload.adminId } returns TestAdministrators.KOBLENZ_PROJECT_ADMIN.id
+
+        transaction {
+            TestData.createApiToken(creatorId = TestAdministrators.KOBLENZ_PROJECT_ADMIN_2.id)
+
+            val tokenBefore = ApiTokens.select { ApiTokens.id eq 1 }.count() > 0
+            assertNotNull(tokenBefore)
+
+            ApiTokenService().deleteApiToken(1, mockDfe)
+
+            val tokenAfter = ApiTokens.select { ApiTokens.id eq 1 }.singleOrNull()
+            assertNull(tokenAfter)
+        }
+    }
+
+    @Test
+    fun deleteApiToken_deletesNoTokenFormOtherProject() {
+        every { mockJwtPayload.adminId } returns TestAdministrators.KOBLENZ_PROJECT_ADMIN.id
+        transaction {
+            TestData.createApiToken(creatorId = TestAdministrators.NUERNBERG_PROJECT_ADMIN.id)
+
+            val numberOfTokensBefore = ApiTokens.selectAll().count()
+            ApiTokenService().deleteApiToken(1, mockDfe)
+            val numberOfTokensAfter = ApiTokens.selectAll().count()
+            assert(numberOfTokensBefore == numberOfTokensAfter)
+        }
+    }
+
+    @Test
+    fun deleteApiToken_throwsForbiddenExceptionWhenNotAuthorized() {
+        every { mockJwtPayload.adminId } returns TestAdministrators.NUERNBERG_PROJECT_STORE_MANAGER.id
+
+        transaction {
+            assertThrows(ForbiddenException::class.java) {
+                ApiTokenService().deleteApiToken(1, mockDfe)
+            }
+            assertFalse(
+                Authorizer.mayAddApiTokensInProject(
+                    AdministratorEntity.find { Administrators.id eq mockJwtPayload.adminId }.single()
+                )
+            )
+        }
+    }
+
+    @Test
+    fun deleteApiToken_throwsExceptionWhenAdminNotFound() {
+        every { mockJwtPayload.adminId } returns 1234
+
+        transaction {
+            assertThrows(ForbiddenException::class.java) {
+                ApiTokenService().deleteApiToken(1, mockDfe)
+            }
+        }
+    }
+
+    @Test
+    fun getApiTokenMetaData_returnsMetaDataSuccessfully() {
+        every { mockJwtPayload.adminId } returns TestAdministrators.KOBLENZ_PROJECT_ADMIN.id
+
+        transaction {
+            TestData.createApiToken(creatorId = TestAdministrators.KOBLENZ_PROJECT_ADMIN.id)
+
+            val metaData = ApiTokenQueryService().getApiTokenMetaData(mockDfe)
+            assertEquals(1, metaData.size)
+            assertEquals(1, metaData[0].id)
+            assertEquals(TestAdministrators.KOBLENZ_PROJECT_ADMIN.email, metaData[0].creatorEmail)
+            assertEquals(LocalDate.now().plusYears(1).toString(), metaData[0].expirationDate)
```

Comment:
```suggestion
            val tokenId = TestData.createApiToken(creatorId = TestAdministrators.KOBLENZ_PROJECT_ADMIN.id)

            val metaData = ApiTokenQueryService().getApiTokenMetaData(mockDfe)
            assertEquals(1, metaData.size)
            assertEquals(tokenId, metaData[0].id)
            assertEquals(TestAdministrators.KOBLENZ_PROJECT_ADMIN.email, metaData[0].creatorEmail)
            assertEquals(LocalDate.now().plusYears(1).toString(), metaData[0].expirationDate)
```
fix for the failing test